apps/coscrad-frontend/src/components/NavBar/NavBar.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { MemoryRouter } from 'react-router-dom';
+import { renderWithProviders } from '../../utils/test-utils';
+import { NavBar } from './NavBar';
+
+describe('NavBar', () => {
+    it('should render successfully', () => {
```

Comment:
You might ask yourself if there's anything more you could test here.

Reply:
Would you create a dummy `navItemInfos` array and then check if the dummy items are displayed?

Reply:
We should have a NavBar container and presenter separation in the future and then implement further tests.